logtail/logtail.go

```diff
@@ -590,5 +591,8 @@ func parseAndRemoveLogLevel(buf []byte) (level int, cleanBuf []byte) {
 	if bytes.Contains(buf, v2) {
 		return 2, bytes.ReplaceAll(buf, v2, nil)
 	}
+	if i := bytes.Index(buf, vJSON); i != -1 {
+		return 1, buf[i+len(vJSON):]
```

Comment:
Why is JSON level 1?

Reply:
So that it goes to logcatcher but not syslog/etc. This helps reduce the volume of logs stored locally, which is an ongoing source of (not unreasonable) complaints.